tests/reftest/mod.rs

```diff
@@ -19,9 +19,14 @@ fn reftest() {
 fn reftest_scaled() {
     let base = &Path::new("tests").join("reftest").join("images");
     reftest_scaled_file(&base.join("rgb.jpg"), 500, 333, &base.join("rgb.png"));
-    reftest_scaled_file(&base.join("rgb.jpg"), 250, 167, &base.join("rgb_250x167.png"));
-    reftest_scaled_file(&base.join("rgb.jpg"), 125, 84,  &base.join("rgb_125x84.png"));
-    reftest_scaled_file(&base.join("rgb.jpg"), 63,  42,  &base.join("rgb_63x42.png"));
+    reftest_scaled_file(
+        &base.join("rgb.jpg"),
+        250,
+        167,
+        &base.join("rgb_250x167.png"),
+    );
```

Comment:
There's must be better algorithm for consistent fmt, I'm sure. We just haven't found it yet. But I shall accept it :joy: